README.md

```diff
@@ -163,16 +163,16 @@ Esta última sesión se iniciará a las 17:00 con una actividad abierta sobre c
 
 Luego, en nuestra última sesión del curso veremos algunos ejemplos adicionales que complementan lo que revisamos a lo largo de esta semana.
 
-* El código que vayamos escribiendo hoy irá apareciendo en [este enlace](https://www.dropbox.com/s/ruupf6x497e9ma7/script_sesion5.R?dl=0). El archivo ya tiene algunas líneas del código de ayer que usaremos para empezar hoy (básicamente, reconstruiremos lo que hicimos al final de la clase de una forma mucho más rápida y eficiente).
+* El código que vayamos escribiendo hoy, irá apareciendo en [este enlace](https://www.dropbox.com/s/ruupf6x497e9ma7/script_sesion5.R?dl=0). El archivo ya tiene algunas líneas del código de ayer que usaremos para empezar hoy (básicamente, reconstruiremos lo que hicimos al final de la clase de una forma mucho más rápida y eficiente).
```

Comment:
No se utiliza coma para separar el sujeto y el verbo de una oración.

Reply:
You are right!